app/(auth)/sign-in/flag-search.tsx

```diff
@@ -0,0 +1,109 @@
+import { router } from 'expo-router'
+import { useCallback, useState } from 'react'
+import { FlatList, ListRenderItem, View } from 'react-native'
+import CountryFlag from 'react-native-country-flag'
+
+import countries from '@/components/controls/country-select/countries.json'
+import { useUsedCountryStorage } from '@/components/controls/country-select/useUsedCountryStorage'
+import TextInput from '@/components/inputs/TextInput'
+import ScreenContent from '@/components/screen-layout/ScreenContent'
+import ScreenView from '@/components/screen-layout/ScreenView'
+import Button from '@/components/shared/Button'
+import DismissKeyboard from '@/components/shared/DissmissKeyboard'
+import FlexRow from '@/components/shared/FlexRow'
+import PressableStyled from '@/components/shared/PressableStyled'
+import Typography from '@/components/shared/Typography'
+import { useTranslation } from '@/hooks/useTranslation'
+
+type Country = {
+  country: string
+  iso: string
+  code: string
+}
+
+const CountrySearchScreen = () => {
+  const t = useTranslation('Common')
+  const [searchedCountry, setSearchedCountry] = useState('')
+
+  const [, setSelectedCountry] = useUsedCountryStorage()
+
+  const handleCancel = useCallback(() => {
+    if (router.canGoBack()) {
+      router.back()
+    } else {
+      router.replace('/sign-in')
+    }
+  }, [])
+
+  const filteredCountries = countries.filter((country) => {
+    return (
+      country.iso.toLowerCase().includes(searchedCountry.toLowerCase()) ||
+      country.country.toLowerCase().includes(searchedCountry.toLowerCase()) ||
+      country.code.toLowerCase().includes(searchedCountry.toLowerCase())
+    )
+  })
+
+  const handleChangeText = (value: string) => {
+    setSearchedCountry(value)
+  }
+
+  const renderItem: ListRenderItem<Country> = useCallback(
+    ({ item }) => {
+      return (
+        <PressableStyled
+          onPress={() => {
+            setSelectedCountry(item.iso)
+            router.back()
+          }}
+          className="border-b border-divider py-4"
+        >
+          <FlexRow className="items-center g-3">
+            <View className="ml-1 h-6 w-6 items-center justify-center overflow-hidden rounded-full">
```

Comment:
Pleas follow design using `p-1` instead of `ml-1`
![image](https://github.com/bratislava/paas-mpa/assets/19418224/e5f7a814-b55f-4815-8f62-dac8a299f73d)